src/test/java/seedu/recipe/testutil/RecipeBuilder.java

```diff
@@ -125,6 +129,11 @@ public RecipeBuilder withGrains(String ... grains) {
      * the {@code Recipe} that we are building.
      */
     public RecipeBuilder withVegetables(String ... vegetables) {
+        /*        if (isDefaultGrainPresent) {
```

Comment:
Just to clarify, are we leaving these comments here or is it temporary? :)

Reply:
hey! i commented those out as i was still trying to fix the failing test case for making the ingredients field compulsory.
Ive fixed it in the next commit and have uncommented these blocks of code :)